common/reviews/api/api-extractor.api.ts

```diff
@@ -1,3 +1,4 @@
+// @public
```

Comment:
Why do you need to change those public/internal api for api-extractor to move a project?